src/main/java/org/folio/processing/mapping/mapper/reader/record/marc/MarcRecordReader.java

```diff
@@ -258,11 +258,28 @@ private String retrieveStringFromLastBrackets(String mappingParameter) {
   }
 
   private String retrieveStringWithBracketsFromLastOne(String mappingParameter) {
+    if (mappingParameter.indexOf(FIRST_BRACKET) > mappingParameter.indexOf(SECOND_BRACKET))
+      return mappingParameter.substring(mappingParameter.indexOf(FIRST_BRACKET), mappingParameter.lastIndexOf(SECOND_BRACKET) + 1);
     return mappingParameter.substring(mappingParameter.indexOf(FIRST_BRACKET), mappingParameter.indexOf(SECOND_BRACKET) + 1);
   }
 
   private String retrieveNameWithoutCode(String mappingParameter) {
```

Comment:
@JavokhirAbdullayev  I think this method-name should be improved. Smth like "retrieveNameWithoutBrackets" or "retrieveNameIgnoringBrackets"